features/upgrade/sdn/sdn2ovn-migration.feature

```diff
@@ -40,7 +40,6 @@ Feature: sdn2ovn migration testing
   @upgrade-prepare
   @vsphere-ipi
   @vsphere-upi
-  @qeci
```

Comment:
Any context for the change ?

Reply:
I saw the logs show 

```
+ cucumber --tags '@4.15 and @amd64 and @vsphere-ipi and @network-openshiftsdn and not @fips and not @customer
        and not @destructive
        and not @flaky
        and not @inactive
        and not @prod-only
        **and not @qeci**
        and not @security
        and not @stage-only
```

https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-vsphere-ipi-sdn-migration-ovn-f14/1814602491191365632/artifacts/vsphere-ipi-sdn-migration-ovn-f14/cucushift-upgrade-prepare/build-log.txt

Reply:
Yes, test cases tagged with qeci means it can only execute on Jenkins CI, so they are exclude in Prow CI.
So those two tags are mistakenly added in the initial PR, or is there something change recently ?

Reply:
this case can be tested in Prow CI,  so `qeci` tag should not be for this case.  
no changes recently for this feature,  I guess it's one mistake at the begin of PR https://github.com/openshift/verification-tests/pull/3182

Reply:
Ack